controllers/helmchart_controller.go

```diff
@@ -991,6 +1034,15 @@ func observeChartBuild(obj *sourcev1.HelmChart, build *chart.Build, err error) {
 		conditions.Delete(obj, sourcev1.BuildFailedCondition)
 	}
 
+	if build.VerificationSignature != nil && build.ProvFilePath != "" {
+		var sigVerMsg strings.Builder
+		sigVerMsg.WriteString(fmt.Sprintf("chart signed by: %v", strings.Join(build.VerificationSignature.Identities[:], ",")))
```

Comment:
```suggestion
		sigVerMsg.WriteString(fmt.Sprintf("chart signed by: '%s'", strings.Join(build.VerificationSignature.Identities[:], ",")))
```
Would this be better? We usually put such information that can be highlighted in quotes.

But overall, I'm not sure about such a long condition message. But we don't have anywhere else to put this info.
```
chart signed by: TestUser using key with fingeprint: 943CB5929ECDA2B5B5EC88BC7035BA97D32A87C1 and hash verified: sha256:007c7b7446eebcb18caeffe9898a3356ba1795f54df40ad39cfcc7382874a10a
```
from the associated test.

Reply:
The message itself isn't that long, it's the data that's taking up quite a bit of space. But as you said, we can't really put this information anywhere else.